client/src/features/form/UserForm.js

```diff
@@ -1,40 +1,119 @@
-import { useState } from "react";
+import { useState, useEffect } from "react";
+
+// Form components
 import { FormMoverContext } from "./contexts/FormMoverContext";
 import FormMover from "./FormMover";
 import FormMoverControl from "./FormMoverControl";
 import FormCreateEvent from "./FormCreateEvent";
 import FormScheduleEvent from "./FormScheduleEvent";
 import FormConfirm from "./FormConfirm";
 import FormSuccess from "./FormSuccess";
+
+// Import data via API for some reason?
 import DataService from "services/dataService";
+
+
 import { generateRecurringDatesArray } from "utilities/calendar";
 
+// This is the code for the form where you add events to the calendar
 const UserForm = () => {
+
+  // Keep track of what "step" we're on in the form  
   const [currentStep, setCurrentStep] = useState(1);
 
+  // This keeps track of the data the user is creating while the form is being filled out
   const [userData, setUserData] = useState({
     recurring: { rate: "noRecurr", days: [] },
   });
+
+  /* Notes about how the form works:
+  Start Date is the default date for a one-off singular event
+  End Date is a date you can set IF you are recurring.
+  */
+
+  // ISAAC'S ISSUE TO SOLVE:
```

Comment:
@isaaclee12 I'm ok with all this for now, but the finished PR into development will need to have this cleaned up.

Reply:
Noted